### PR TITLE
Adding jobflow templates

### PR DESCRIPTION
**Pull request by pro-akim**
# Description

Jobflow examples added in template folder

## Test results:

CC :green_circle: 
[central.txt](https://github.com/user-attachments/files/16635138/central.txt)

Manager :green_circle: 
[manager.txt](https://github.com/user-attachments/files/16635139/manager.txt)

Agent :green_circle: 
[agent.txt](https://github.com/user-attachments/files/16635137/agent.txt)

### CODE REVIEW

File: deployability/modules/jobflow/templates/central_components/Test-CC-complete.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+version: 0.1
+description: This workflow is used to test the Wazuh manager deployment for DDT1 PoC
```
Comment (fcaffieri) [inline deployability/modules/jobflow/templates/central_components/Test-CC-complete.yaml:2]: Same

File: deployability/modules/jobflow/templates/manager/Test-manager-complete.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+version: 0.1
+description: This workflow is used to test manager deployment for DDT1 PoC
```
Comment (fcaffieri) [inline deployability/modules/jobflow/templates/manager/Test-manager-complete.yaml:2]: Same

**Review by fcaffieri — CHANGES_REQUESTED** [review]

File: deployability/modules/jobflow/templates/central_components/Test-CC-complete.yaml (hunk 1 above)
Comment (pro-akim) [reply deployability/modules/jobflow/templates/central_components/Test-CC-complete.yaml]: Done https://github.com/wazuh/wazuh-qa/pull/5680/commits/0424a1f8a05fb3f390a6ee3bc9c55d0021f3fef8

File: deployability/modules/jobflow/templates/agent/Test-agents-complete.yaml [hunk 3]
```diff
@@ -0,0 +1,259 @@
+version: 0.1
+description: This workflow is used to test agents deployment for DDT1 PoC
```
[the remark this answers is not in the document]
Comment (pro-akim) [reply deployability/modules/jobflow/templates/agent/Test-agents-complete.yaml]: Done https://github.com/wazuh/wazuh-qa/pull/5680/commits/0424a1f8a05fb3f390a6ee3bc9c55d0021f3fef8

File: deployability/modules/jobflow/templates/agent/Test-agents-complete.yaml [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+          - label-team: "qa"
+          - label-issue: "https://github.com/wazuh/wazuh/issues/<issue_number>"
+    cleanup:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: delete
+          - track-output: "{working-dir}/manager-{manager-os}/track.yaml"
```
[the remark this answers is not in the document]
Comment (pro-akim) [reply deployability/modules/jobflow/templates/agent/Test-agents-complete.yaml]: Done https://github.com/wazuh/wazuh-qa/pull/5680/commits/0424a1f8a05fb3f390a6ee3bc9c55d0021f3fef8

**Review by fcaffieri — APPROVED** [review]
LGTM

**Pull request merged**